addons/crm_livechat/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'CRM Livechat',
+    'category': 'crm',
+    'summary': 'Create lead from livechat conversation',
+    'depends': [
+        'crm',
+        'im_livechat'
+    ],
+    'description': 'Create new leads with ease using commands in the channel',
```

Comment:
Sounds strange. Furthermore, we don't use commands, we use only one command. Explain what command we define in this module.